stdnum/md/__init__.py

```diff
@@ -19,3 +19,6 @@
 # 02110-1301 USA
 
 """Collection of Moldavian numbers."""
+
+# provide aliases
+from stdnum.md import idno as vat  # noqa: F401
```

Comment:
According to https://support.wix.com/en/article/about-vat-in-moldova the VAT number consists of 7 digits (with an optional MD prefix) which doesn't match the format of the IDNO.